src/app/aircraft/add/post.controller.js

```diff
@@ -14,18 +17,16 @@ module.exports = (req, res) => {
   const craftBase = _.toUpper(req.body.craftBase);
 
   // Define a validation chain for user registeration fields
-  const craftRegChain = [
```

Comment:
Rather than duplicate the validations, take the ones from the GAR form aircraft screen, as not only are they the same, the GAR form one contains the additional validations for string lengths.